spec/lib/open_food_network/order_and_distributor_report_spec.rb

```diff
@@ -34,9 +34,9 @@ module OpenFoodNetwork
       end
 
       it "should denormalise order and distributor details for display as csv" do
-        subject = OrderAndDistributorReport.new [@order]
+	subject = OrderAndDistributorReport.new create(:admin_user), {}, true
 
-        table = subject.table
+        table = subject.send(:line_item_details, [@order])
```

Comment:
This was just the result of refactoring and I'm not sure how to resolve that. I don't think we should change the method to public just so that we can test it. Strictly speaking, we just want to test the public methods, but I would have needed to rewrite the test for that (more effort). What would you do?

Reply:
I don't want to make it public either. I wonder if we can still rely on the `#table` method to compare the expected output against.